libres/lib/python/init.cpp

```diff
@@ -0,0 +1,22 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <vector>
+#include <iostream>
+#include <ert/enkf/obs_vector.hpp>
+
+namespace py = pybind11;
+
+obs_vector_type* to_obs_vector_type(py::object obj) {
+    py::object address = obj.attr("_address")();
```

Comment:
https://github.com/equinor/cwrap/blob/2508302d089ff5733da3d3241ed49c17e9ceb33f/cwrap/basecclass.py#L48-L49

This we can do directly to avoid a function call into Python. (minor optimisation)

```suggestion
    py::object address = obj.attr("_BaseCClass__c_pointer");
```